macro-plantuml-macro/src/main/java/org/xwiki/contrib/plantuml/internal/DefaultPlantUMLGenerator.java

```diff
@@ -57,6 +57,12 @@ public class DefaultPlantUMLGenerator implements PlantUMLGenerator
 
     @Override
     public void outputImage(String input, OutputStream outputStream, String serverURL) throws IOException
+    {
+        outputImage(input, outputStream, serverURL, "png");
```

Comment:
This looks like some unnecessary code duplication since we allow to pass null to default to the default configuration.